pwndbg/gdblib/disasm/instruction.py

```diff
@@ -81,6 +81,10 @@ class InstructionCondition(Enum):
     UNDETERMINED = 3
 
 
+def boolean_to_instruction_condition(condition: bool) -> InstructionCondition:
+    return InstructionCondition.TRUE if condition else InstructionCondition.FALSE
```

Comment:
Honestly, I'm not sure I like this enum overall. I understand we want to encode three values but it seems like `True, False, None` would do the same job.

Reply:
I think treating a `bool` type as three distinct values is just asking for bugs (if x vs is x if not None). I also have plans for this enum to add a fourth value - `InstructionCondition.UNCONDITIONAL`, allowing for more preciseness on the type of instruction we are dealing with. Currently, `UNDETERMINED` is very vague.

Reply:
> I think treating a bool type as three distinct values is just asking for bugs (if x vs is x if not None)

Yes, but this is why we almost never use the former syntax (it sometimes sneaks in when I can't get to a code review, I'll check if we can add a lint for it).